storage/client/queries/queries.go

```diff
@@ -462,10 +462,11 @@ const (
 		WHERE
 			(tokens.runtime = $1) AND
 			($2::oasis_addr IS NULL OR tokens.token_address = $2::oasis_addr) AND
+			($3::text IS NULL OR tokens.token_name ILIKE '%' || $3 || '%') AND
```

Comment:
the name input is not escaped, searches with `%` and `_` will do special things

Reply:
The clickup issue also mentioned filtering by ticker (`symbol` in the db).

Reply:
oh let me add that

Reply:
added same stuff for symbol